editor/components/post-title/index.js

```diff
@@ -99,12 +99,15 @@ class PostTitle extends Component {
 						'mod+shift+z': this.redirectHistory,
 					} }
 				>
+					<label htmlFor="post-title" className="screen-reader-text">
+						{ placeholder || __( 'Add title' ) }
+					</label>
 					<Textarea
+						id="post-title"
```

Comment:
Even if it's something we assume to be used once, having a static `id` feels like a poor practice in any React component, and I'd think this should still use `withInstanceId` to guarantee a unique ID.